### ISSUE TITLE
Unable to run test android in cli 8.1.1

### ISSUE THREAD

**Issue by jcassidyav** [NativeScript/nativescript-cli#5572]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 8.1.1
 - Cross-platform modules:
 - Android Runtime:
 - iOS Runtime:
 - XCode Version:
 - Plugin(s):

**Describe the bug**
Running ns test android results in the error

```
TypeError: Cannot read property 'hostname' of null
    at HttpClient.<anonymous> (C:\Users\jcassidy\AppData\Roaming\npm\node_modules\nativescript\lib\common\http-client.js:77:44)
    at Generator.next (<anonymous>)
    at fulfilled (C:\Users\jcassidy\AppData\Roaming\npm\node_modules\nativescript\lib\common\http-client.js:5:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)```
```
Also when this is fixed running tests results in an error because the socket.io is not downloaded and saved correctly.

**To Reproduce**
execute ```ns test android``` on a project with tests configured that worked with cli 8.02. 

**Expected behavior**
Tests should execute.

**Sample project**

**Additional context**

**Comment by EdJones** [comment]
With ns 8.1.1, I am getting the `Cannot read property 'hostname' of null` as a cli error when running `ns publish iOS`.

**Comment by rigor789** [comment]
@EdJones #5573 should fix that as well, I'll bump a patch release shortly.

**Comment by rigor789** [comment]
Published 8.1.2 with the fix.
